lib/textrazor/phrase.rb

```diff
@@ -5,8 +5,8 @@ class Phrase
     attr_reader :id, :text
 
     def initialize(params, words)
-      @id    = params["id"]
-      @text  = match_words(params["wordPositions"], words)
+      @id    = params[:id]
+      @text  = match_words(params[:wordPositions], words)
```

Comment:
I just had another look at your PR and I think we will be better off using the `initialize_params` method defined in [`util`](https://github.com/andhapp/textrazor/blob/master/lib/textrazor/util.rb)

Will you be happy to make that change?

Reply:
seems it already fixed. 

https://github.com/andhapp/textrazor/blob/master/lib/textrazor/phrase.rb

Reply:
I checked Util, I feel it is hard to understand for something.

1. If it's a utility tool, it will be better for me to use the class method, `TextRazor::Util.set_params_to_vars params`
2. You have a conflicted method `create_from_hash` to this `Phrase` module.
3. Currently what I do really need just a `with_indifferent_access` in active support, but I think to load active support is too much, so we can implement this method here.

What do you think?

Reply:
This code was refactored at some point and I think `Phrase` class was missed. You're right `create_from_hash` is duplicate and I don't think it's needed and can be removed.

Would you be willing to refactor it to replicate how rest of the code looks? I know it's a big ask, if  you're not happy to do, I will do it and ensure it works with both strings and symbols as keys.

Please let me know your thoughts.

Thanks.

Reply:
ok, I just removed the `create_from_hash` method, but for the refactoring, it's not necessary, I actually tried the latest version of textrazor in my project, seems working now. 

Thanks